rasa/cli/run.py

```diff
@@ -82,7 +82,7 @@ def run(args: argparse.Namespace):
         args.credentials, "credentials", DEFAULT_CREDENTIALS_PATH, True
     )
 
-    if args.enable_api:
+    if args.enable_api and not args.remote_storage:
```

Comment:
If you add the condition `not args.remote_storage` to the if statement here, it will not be possible to start the Rasa server with a remote storage and enable the API. Or am I wrong? I would suggest:
```
if args.enable_api:
    if not args.remote_storage:
        args.model = _validate_model_path(args.model, "model", DEFAULT_MODELS_PATH)
    rasa.run(**vars(args))
    return 
```

Reply:
Oh, I see. I need to understand the codebase, but yes it makes sense. :smiley: